tests/e2e/specs/setup-mc/step-1-accounts.test.js

```diff
@@ -616,6 +613,61 @@ test.describe( 'Set up accounts', () => {
 		} );
 	} );
 
+	test.describe( 'Claim Google Ads Account', () => {
+		test.beforeAll( async () => {
+			await setUpAccountsPage.mockJetpackConnected();
+			await setUpAccountsPage.mockGoogleConnected();
+			await setUpAccountsPage.mockMCHasAccounts();
+			await setUpAccountsPage.mockMCConnected();
+			await setUpAccountsPage.fulfillAdsAccounts( { id: 12345 } );
```

Comment:
The payload should be `Array<Object>` type.

Reply:
👍